products/sle/main.pm

```diff
@@ -538,6 +538,7 @@ sub load_baremetal_tests {
     loadtest "autoyast/prepare_profile" if get_var "AUTOYAST_PREPARE_PROFILE";
     if (get_var('IPXE')) {
         loadtest 'installation/ipxe_install';
+        load_inst_tests() if !get_var(AUTOYAST);
```

Comment:
Bareword "AUTOYAST" not allowed while "strict subs" in use at os-autoinst-distri-opensuse/products/sle/main.pm line 541.

Reply:
OK, also unset `AUTOYAST_PREPARE_PROFILE` is needed.

Reply:
I updated the commit message.